src/features/visualizations/Racing-Chart/RacingData.js

```diff
@@ -73,32 +78,71 @@ function RacingData() {
             deaths: x.deaths,
             date: x.date,
           }))
-          .filter((x) => compareAsc(parseISO(x.date), dateToFilter) === 0)
+          .filter(
+            (x) => getDayOfYear(parseISO(x.date)) === getDayOfYear(dateToFilter)
+          )
       );
       setDateToFilter(add(dateToFilter, { days: 1 }));
+
+      if (
+        getDayOfYear(parseISO(deaths[deaths.length - 1].date)) ===
+        getDayOfYear(dateToFilter)
+      ) {
+        return setStart(false);
+      }
     }
   }, 200);
 
   useEffect(() => {
     dispatch(getConfirmedCases());
   }, [dispatch]);
 
+  const handleReset = (e) => {
+    e.preventDefault();
+    setStart(false);
+    setData(
+      deaths
+        .map((x) => ({
+          name: x.country,
+          deaths: x.deaths,
+          date: x.date,
+        }))
+        .filter((x) => x.date === deaths[0].date)
+    );
+    setDateToFilter(new Date(deaths[0].date));
+  };
```

Comment:
This is what we needed to get the reset button working. It resets the filter to the starting conditions set in the useEffect that fires on mount.